### ISSUE TITLE
Reproduction results on PGM dataset are poor

### ISSUE THREAD

**Issue by WuGuommming** [yanivbenny/MRNet#2]
Greetings! I have encountered some problems while using your code to reproduce.

I used your mrnet project on GitHub and trained on RAVEN-FAIR dataset and PGM dataset. RAVEN-FAIR dataset is generated by your another GitHub project, and PGM dataset we used only contains neutral region.

We've tried two ways - training from the beginning and training from your pretrained model.

The results of 
    training from the beginning in RAVEN-FAIR
    pretrained model in RAVEN-FAIR
    pretrained model in PGM
are almost the same as those in the paper.
However, the result of training from beginning in PGM dataset is very poor, and the accuracy is only 54.14%. Therefore, I would like to ask you the possible reasons, especially whether there are some problems with parameters?

For the two datasets, except that the WD you mentioned on GitHub is 1e-6 and 1e-5 respectively, we have not modified other parameters and use the default values in the project. The specific parameters when we train PGM dataset are as follows:
beta1=0.9
beta2=0.999
contrast=False
dataset='PGM'
debug=False
dropout=False
early_ stopping=10
epochs=-1
epsilon=1e-08
flip=False
force_ bias=False
img_ size=80
levels='111'
loss_ func='contrast'
lr=0.001
meta_ beta=0.0
model_ name='mrnet'
multihead=True multihead_ mode=None
multihead_ w=1.0
r_ func='dist'
ratio=None, recovery=False
regime=None
relu_ before_ reduce=False
row_ col=True, seed=12345
subset=None
test=False
testname='PGM'
use_ tag=1
wd=1e-06
weighted_ loss=False

I would appreciate your help:）

**Comment by yanivbenny** [comment]
Sorry for the late response.

The PGM pretrained model is accidentally one of the ablation models.
We did not keep other checkpoints so it's impossible to retrieve the original model.

As for the training results, we are looking at the cause of this error.

**Comment by yanivbenny** [comment]
Thank you for raising this issue.
The training details have been updated and a new model has been uploaded which achieved 93% on the test set.
